ScoutSuite/providers/aws/provider.py

```diff
@@ -104,6 +106,25 @@ def _add_security_group_name_to_ec2_grants(self):
                            self.add_security_group_name_to_ec2_grants_callback,
                            {'AWSAccountId': self.aws_account_id})
 
+    def _add_security_group_data_to_elbv2(self):
```

Comment:
This method is a bit cryptic. One way of making it more readable would be to iterate through the values instead of the keys. Also, using words instead of acronyms. For example:

```py
for region in elbv2_config['regions'].values():
    for vpc in region['vpcs'].values():
        for load_balancer in vpc['lbs'].values():
            for i in range(len(load_balancer['security_groups'])):
                for security_group in vpc['security_groups'].values():
                    try:
                        if load_balancer['security_groups'][i]['GroupId'] == sg:
                             load_balancer['security_groups'][i] = vpc['security_groups'][sg]
                    except KeyError:
                        pass

                    check_security_group_rules(load_balancer, i, 'ingress')
                    check_security_group_rules(load_balancer, i, 'egress')
```

I'm not sure there's a better way of implementing this, but the five levels of nesting have a bit of a bad smell smell. Is it possible that some users might have a large amount of vpcs, load balancers and security groups? *O(n^4)*, or at least *O(n_vpcs * n_lbs * n_sgs_in_lb * n_sgs_in_vpc)*, considering the number of regions is constant.